src/flavors/creatingcascadia/config.json

```diff
@@ -273,15 +273,15 @@
           {
             "name": "title",
             "type": "text",
-            "prompt": "How does it feel?",
+            "prompt": "What's happening here?",
             "display_prompt": " ",
             "placeholder": "Title here...",
             "optional": false
           },
           {
             "name": "description",
             "type": "rich_textarea",
-            "prompt": "What is the aesthetic of this place?",
+            "prompt": "What is it, and how it can it change things?",
```

Comment:
@goldpbear I figured these will be reasonable prompts that will work across the sensation/idea/artwork/event location types.

Reply:
That seems reasonable-- I'm not up to speed on the specifics of the language here, but I think some minor changes are probably ok.